src/interfaces/IMorpho.sol

```diff
@@ -295,6 +295,9 @@ interface IMorpho {
     /// @param signature The signature.
     function setAuthorizationWithSig(Authorization calldata authorization, Signature calldata signature) external;
 
+    /// @notice Accrues interest for `marketParams`.
+    function accrueInterest(MarketParams memory marketParams) external;
```

Comment:
The function misses the natspec doc for the `marketParams` input parameter

Reply:
We won't add it for simplicity